index.html

```diff
@@ -16,5 +16,19 @@
       </p>
       <button class="button button-type-1">watch online</button>
     </div>
+
+
+    <div class="old-man">
+      <h2 class="text header-2">The Backstage of the Wilderness World.</h2>
+      <img class="img-men" src="./assets/images/bamboo_cap.png">
+      <span class="text body">The site was founded on the basis of a volunteer movement to protect and care for animals.</span>
+      <span class="text subtitle">How it works</span>
+      <span class="text body-medium">The main goal is to help the animals, as well as the nature reserves and zoos where they are kept. We are currently working on video projects targeting pandas in China, eagles on an island near Los Angeles, alligators in Florida and gorillas in the Congo. These have a total of more than 1,500 mammals and reptiles.</span>
+    </div>
+    <div>
```

Comment:
Why do you need this div? There are no styles, so probably you can remove this div